src/styles/OurTeamPage.module.less

```diff
@@ -0,0 +1,44 @@
+.ourteampage {
+  width: 100vw;
+  text-align: center;
+  padding: 10%;
```

Comment:
I think this might be too much padding. It looks like quite a bit on my screen. Could you try maybe 5%